src/applications/personalization/connected-accounts/components/ConnectedApps.jsx

```diff
@@ -47,8 +47,9 @@ export function ConnectedApps({ confirmDelete, propertyName, accounts }) {
               })
             }
           >
-            Go to {propertyName} FAQs
+            Go to connect account FAQs
```

Comment:
Is this right?

Reply:
It was supposed to be "connected account". I just pushed a fix. Thanks for catching that.